### PR TITLE
feat: pass base URL and subscription key as input parameter in instance creation (SDKCF-3937)

### PR DESCRIPTION
**Pull request by imaginaris**
# Description
Config parameters are now passed directly as init arguments instead of putting them in Info.plist.
In order to use the module, user must create an instance of `RSignatureVerifier` and call `verify()` instance method.

## Links
SDKCF-3937

# Checklist
- [X] I have read the [contributing guidelines](../CONTRIBUTING.md).
- [X] I wrote/updated tests for new/changed code
- [X] I removed all sensitive data before every commit, including API endpoints and keys
- [X] I ran `fastlane ci` without errors

### CODE REVIEW

**Comment by imaginaris** [comment]
The `configure-secrets.sh` script is not needed now. Should I remove it or disable it?

**Comment by apparition47** [comment]
IMO okay to remove since it can be dug up in git history anyway

File: Tests/Integration/IntegrationTests-Info.plist [hunk 1]
```diff
@@ -20,6 +20,10 @@
 	<string>????</string>
 	<key>CFBundleVersion</key>
 	<string>1</string>
+	<key>RAS_SUBSCRIPTION_KEY</key>
+	<string>$(RAS_SUBSCRIPTION_KEY)</string>
+	<key>RSV_KEY_FETCH_ENDPOINT</key>
+	<string>$(RSV_KEY_FETCH_ENDPOINT)</string>
```
Comment (donnie-jp) [inline Tests/Integration/IntegrationTests-Info.plist:26]: Still needed?

**Review by donnie-jp — APPROVED** [review]
LGTM

**Review by vincentdelmundo — APPROVED** [review]
LGTM

File: RSignatureVerifier/RSignatureVerifier.swift [hunk 2]
```diff
@@ -1,20 +1,31 @@
 /// Signature Verifier public API functions
 @objc public final class RSignatureVerifier: NSObject {
 
+    let realInstance: RealSignatureVerifier
+
+    /// Initializes new instance configured with given parameters
+    /// - Parameter baseURL: URL used to fetch and store public key data
+    /// - Parameter subscriptionKey: key used to authenticate request to `baseURL` endpoint
+    @objc public init(baseURL: URL, subscriptionKey: String) {
+        realInstance = RealSignatureVerifier(fetcher: Fetcher(apiClient: APIClient(),
```
Comment (malayli) [inline RSignatureVerifier/RSignatureVerifier.swift:10]: Is `realInstance` preferred than `self`?

Comment (imaginaris) [reply RSignatureVerifier/RSignatureVerifier.swift:10]: but `self` is not a `RealSignatureVerifier` class/type. I'm not sure what do you mean

**Review by malayli — APPROVED** [review]

**Pull request merged**